frontend/integration-tests/views/crud.view.ts

```diff
@@ -68,10 +69,14 @@ export const filterForName = async (name: string) => {
 };
 
 const actionOnKind = (action: string, kind: string) => {
-  return `${action} ${kind}`;
+  const humanizedKind = (kind.includes('~') ? kind.split('~')[2] : kind)
+    .split(/(?=[A-Z])/)
+    .join(' ');
+
+  return `${action} ${humanizedKind}`;
 };
-export const editKind = (kind: string) => actionOnKind(actions.edit, kind);
```

Comment:
are you sure no other test use this consts ?

Reply:
Didn't touch this, don't know how this got to PR